src/System.Private.Xml/src/System/Xml/Core/XmlTextReaderImpl.cs

```diff
@@ -2579,7 +2574,7 @@ private bool IsResolverNull
         {
             get
             {
-                return _xmlResolver == null || (LocalAppContextSwitches.ProhibitDefaultUrlResolver && !_xmlResolverIsSet);
+                return _xmlResolver == null || !_xmlResolverIsSet;
```

Comment:
Just so I understand, we're assuming now that ProhibitDefaultUrlResolver is true?